### ISSUE TITLE
Dexed VST popups shown in Windows Taskbar

### ISSUE THREAD

**Issue by MD-78** [asb2m10/dexed#268]
1. Load dexed VST3 in REAPER
2. Click dexed logo or PARAM

Problem: Popups are shown in Windows Taskbar. The dexed logo also is shown in front of other Windows programs then.

Expected: No additional Windows Taskbar entries.
![Zwischenablage-1](https://user-images.githubusercontent.com/17949026/106007202-56b5c700-60b6-11eb-9a1f-5a8963a4a206.png)
![Zwischenablage-3](https://user-images.githubusercontent.com/17949026/106007214-587f8a80-60b6-11eb-815a-5cf6fe5a9d1d.png)

dexed 0.9.5 VST3
Windows 8.1 Pro
REAPER 6.21 x64

**Comment by asb2m10** [comment]
Yes, the Dialog are external windows and it should not be the case. Will change this for the next version but need some refactoring to do so.

**Comment by MD-78** [comment]
Cool, thank you!
